src/main/java/seedu/address/model/person/Status.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a Person's status in the address book.
+ * Guarantees: immutable;
+ */
```

Comment:
HealthStatus can be defined as {PENDING, SICK, HEALTHY} to better reflect the Status of Patients